app/routes/utils/user-needs-notification.js

```diff
@@ -0,0 +1,23 @@
+const { multiSpecies } = require('../../config')
+
+const userNeedsNotification = (applications, claims) => {
+  const releaseDate = new Date(multiSpecies.releaseDate).getTime()
+
+  const [latestApplication] = applications
```

Comment:
Though this works, and it works below for the latestClaim, I dont think its super obvious that this is using the first member of the applications array, and ignoring the other ones - it could maybe be more explicit if we did applications[0] ? This is minorly nitpicky though, it might just be obvious regardless. Up to you on this.

On that note, how confident do we feel that the first member of the applications array and the first member of the claims array are the latest ones? Are they sorted before they are passed to this function?

Reply:
I think array destructuring is pretty idiomatic these days? I guess it doesn't really matter when accessing only a single item from an array but I find it slightly more syntactically pleasing personally.

The arrays are pre sorted... [twice in this particular case](https://github.com/DEFRA/ffc-ahwr-application/blob/main/app/repositories/application-repository.js#L65-L68) 😞

Reply:
I agree with Rob it's not that obvious, although array destructuring is fine in general, when used to just grab the first element like this I think it's less obvious than using blah[0], but it's not the end of the world.